packages/manager/modules/hub/src/index.js

```diff
@@ -1,15 +1,25 @@
 import angular from 'angular';
-
 import '@ovh-ux/manager-core';
 import '@uirouter/angularjs';
-import 'angular-translate';
+import 'oclazyload';
 
 const moduleName = 'ovhManagerHub';
 
-angular.module(moduleName, [
-  'ovhManagerCore',
-  'pascalprecht.translate',
-  'ui.router',
-]);
+angular
+  .module(moduleName, ['oc.lazyLoad', 'ui.router', 'ovhManagerCore'])
+  .config(
+    /* @ngInject */ ($stateProvider) => {
+      $stateProvider.state('hub.**', {
```

Comment:
This seems redundant with the manager-hub app routing. Moreover why not target `/`, I'm not sure a `/hub` is needed here. Imo I'm not sure the module hub should contain any layout nor routing, I think it should just contains reusable components and tiles and all of it would be displayed and assembled in the app.

Reply:
The redundant route and html have been removed from the module